examples/infra/eks-bootstrap/variables.tf

```diff
@@ -0,0 +1,6 @@
+variable "cluster_name" {
```

Comment:
Was removing the `create_bastion` variable intentional?

Reply:
Yes - this was a combination of several factors:
* Discussion with Jon on what is a useful "base".
* Generally a move towards being more opinionated in the example configurations, for the following reasons:
  * Less surface area to test
  * There's not much of a downside given that `eks-bootstrap` is now small and easily customizable.
* Consistency with e.g. no `install_multus`, `install_ebs_csi_plugin` variables.

HST I don't mind that much and happy to revert.

Reply:
That sounds fine - just checking it wasn't just missed in the move.